### DISCUSSION TITLE
Modify the welcome page based on security roles.

### DISCUSSION THREAD

**Discussion by kanchansapkota27** [apache/superset#14230] [Q&A / Help]
Is there a way to modify the superset welcome page such as the admin can view the default welcome page while other roles may be able to view things like list of dashboards only. Removing things like recent and logs.

**Comment by suddjian** [comment]
I don't think this is possible, currently. Maybe @pkdotson can confirm? But I think this is a good suggestion! If someone opens a PR I'll be happy to review it!

**Reply by kanchansapkota27** [reply]
Hope this gets implemented in future releases.
